backend/server.js

```diff
@@ -41,17 +41,16 @@ const hapi = new Hapi.Server({
 sbp('okTurtles.data/set', SERVER_INSTANCE, hapi)
 
 sbp('sbp/selectors/register', {
+  'backend/server/broadcastEntry': async function (entry: GIMessage) {
+    console.log(chalk.blue.bold(`[pubsub] Broadcasting ${entry.description()}`))
+    const pubsub = sbp('okTurtles.data/get', PUBSUB_INSTANCE)
+    const pubsubMessage = createMessage('entry', entry.serialize())
+    const subscribers = pubsub.enumerateSubscribers(entry.contractID())
+    await pubsub.broadcast(pubsubMessage, { to: subscribers })
+  },
   'backend/server/handleEntry': async function (entry: GIMessage) {
-    const contractID = entry.contractID()
     await sbp('gi.db/log/addEntry', entry)
-    console.log(chalk.blue.bold(`broadcasting ${entry.description()}`))
-    sbp('okTurtles.data/apply', PUBSUB_INSTANCE, pubsub => {
-      if (contractID in pubsub.subscribersByContractID) {
-        pubsub.subscribersByContractID[contractID].forEach((client) => {
-          client.send(createMessage('entry', entry.serialize()))
-        })
-      }
-    })
+    await sbp('backend/server/broadcastEntry', entry)
```

Comment:
Oh wait nm this was removed later on with a function call. Oh well :)

Reply:
I've added it back! 😄 

However I'm not sure whether the `console.log()` call should take place on the first or the fourth line.

Reply:
@snowteamer well, the only reason to have it is if you'd like to explicitly call `entry.description()` in the logging. The advantage of the SBP selector is that logging happens automatically, but it it won't result in a call to `entry.description()`. Is it possible for JS objects to specify how they are logged to the console? If so, then we could have it so that `this.description()` is called whenever they're logged

Reply:
> Is it possible for JS objects to specify how they are logged to the console?

Yes it is possible in the server console by defining a [`Symbol.for('nodejs.util.inspect.custom')`](https://nodejs.org/api/util.html#util_util_inspect_custom) method on said JS objects.
The colors and formatting can also be specified. We could possibly use it to improve logging for many objects.

Reply:
Awesome, great find! Go for it! 😄

Reply:
However, the nested SBP calls cause redundant logging. I've tried to mitigate this by adding the `'backend/server/broadcastEntry'` selector to the `dontLog` object.

Reply:
Ah, yeah didn't realize that would be a problem, but it makes sense. Good call on adding it to the `dontLog` object. 👍